torch/_inductor/utils.py

```diff
@@ -310,7 +310,7 @@ def timed(model: Callable[..., Any], example_inputs, times: int = 1) -> float:
 
 def print_performance(fn, args=(), times=10, repeat=10, baseline=1.0):
     timings = torch.tensor([timed(fn, args, times) for _ in range(repeat)])
-    took = torch.median(timings)
+    took = torch.median(timings)/times
```

Comment:
Mediand divided by times??

Reply:
yeah, the benchmark runs the function 10 times and takes the overall runtime.

The median is computed over the "repeat" dimension, if that makes sense.

Reply:
Ah, I see. Fair enough.